frontend/src/components/Header.jsx

```diff
@@ -30,6 +31,8 @@ const StyledLink = styled(Link)`
 `;
 
 function Header() {
```

Comment:
The reason why Header is not change Profile to Sign In after logging out is because the Header is not being rerendered after the button click. I could not find a nice way to do this so we can talk in class about work arounds